rbac/migration_tool/migrate.py

```diff
@@ -61,18 +63,27 @@ def get_kessel_relation_tuples(
         }
 
         for perm in v2_role_binding.role.permissions:
-            relationships.append(create_relationship("role", v2_role_binding.role.id, "user", "*", perm))
+            relationships.append(
+                create_relationship(("rbac", "role"), v2_role_binding.role.id, ("rbac", "user"), "*", perm)
+            )
         for group in v2_role_binding.groups:
             # These might be duplicate but it is OK, spiceDB will handle duplication through touch
-            relationships.append(create_relationship("role_binding", v2_role_binding.id, "group", group.id, "subject"))
+            relationships.append(
+                create_relationship(
+                    ("rbac", "role_binding"), v2_role_binding.id, ("rbac", "group"), group.id, "subject"
+                )
+            )
 
         for bound_resource in v2_role_binding.resources:
```

Comment:
AttributeError: 'V2rolebinding' object has no attribute 'resources'

(I know it is draft I just found it as I did rebase)

Reply:
Fixed, ty! (updated a lot of the migrate stuff substantially in recent commits)